common/customizable_schema.go

```diff
@@ -30,6 +90,16 @@ func CustomizeSchemaPath(s map[string]*schema.Schema, path ...string) *Customiza
 	return &CustomizableSchema{Schema: sch, path: path}
 }
 
+func (s *CustomizableSchema) SchemaPath(path ...string) *CustomizableSchema {
+	sch := MustSchemaPath(s.GetSchemaMap(), path...)
+	return &CustomizableSchema{Schema: sch, path: path, context: s.context}
+}
+
+func (s *CustomizableSchema) GetSchemaMap(path ...string) map[string]*schema.Schema {
```

Comment:
`path` parameter is not used. This probably should not be here.

Reply:
Ahh yeah my bad, copy pasta